suricata.yaml.in

```diff
@@ -71,6 +71,10 @@ stats:
   # Add stream events as stats.
   #stream-events: false
 
+# Plugins -- Experimental -- specify the plugin name and filename of the plugin shared object
+plugins:
+#   plugin-name: /path/to/plugin.so
```

Comment:
With the current code this should be a list:
```
plugins:
  - /path/to/plugin.so
  - /path/to/plugin_directory
```
I guess its just by chance that iterating an object or list is more or less the same with our YAML suppose, but that may not always be the case.

Reply:
Will update.